### PR TITLE
Address lingering review comments from SHM locking PR

### PR DESCRIPTION
**Pull request by mheon**
Should also have a fix for a nasty test flake

### CODE REVIEW

**Comment by mheon** [comment]
Followon to #1235

**Comment by baude** [comment]
@rhatdan ptal

File: libpod/lock/shm_lock_manager_linux.go [hunk 1]
```diff
@@ -60,6 +61,11 @@ func (m *SHMLockManager) RetrieveLock(id uint32) (Locker, error) {
 	lock.lockID = id
 	lock.manager = m
 
+	if id >= m.locks.GetMaxLocks() {
+		return nil, errors.Errorf("given lock ID %d is too large - this manager only supports lock indexes up to %d",
```
Comment (rhatdan) [inline libpod/lock/shm_lock_manager_linux.go]: Shorten message

"Lock id %d is too large - manager support max %d lock indexes"

Also wrap 
errors.Wrapf(syscall.EINVAL, "Lock id %d is too large - manager support max %d lock indexes"

Comment (rhatdan) [reply libpod/lock/shm_lock_manager_linux.go]: @mheon Fix this and we can get this merged.

Comment (mheon) [reply libpod/lock/shm_lock_manager_linux.go]: Done

**Comment by mheon** [comment]
bot, retest this please

**Pull request merged**